src/status_im2/contexts/quo_preview/code/snippet.cljs

```diff
@@ -26,6 +26,23 @@
         s.logger.Error(\"failed to set Server.port\", zap.Error(err))
         return
     }
+
```

Comment:
Just added more lines to the example, they are the same as those shown in designs